main.tex

```diff
@@ -18,6 +18,33 @@
 \begin{document}
 
 \maketitle
+\subsubsection {Facebook}
+Eine Studie von \textcite{empowermentThroughFacebook}, welche zwischen 2011 und 2012 in New Delhi durchgeführt wurde, zeigt, dass Facebook zu der Zeit eine große Rolle in der Gesellschaftsentwicklung spielte. 
+\medskip
+Da zu 2012 eine Regulierung von bis zu 200 SMS pro Tag veröffentlicht wurde(Press Information Bureau,
+Government of India, 2012), stieg das Interesse von Facebook zu der Zeit stark an,
+da Handybesitzer weiterhin unlimitiert mit ihren Freunden kommunizieren wollten.
+Durch das Kennenlernen und Nutzen von Facebook bildete sich eine Gesellschaft, welche bislang der Globalisierung
+durch das Internet ausgeschlossen waren und nun erstmals eine transnationale Identität erstellen konnte.
+\medskip
+Bis 2015 wuchs die Nutzeranzahl von Facebook in Indien geradezu explosionsartig, und mit derzeit 135 Millionen Nutzer waren gerademal 25\% weiblich\autocite{slideshareIndia}. Das liegt vorallem daran, dass vorallem jungen indischen Frauen, aus elterlichen Prinzipien, 
+wenig Freiheit gelassen wurde sich mit Freunden zu treffen, oder gar Technologie
+zu besitzen. Dieser Anteil sollte sich laut Aktivisten in der Zukunft stark ändern, 
+aber zu 2018 sind sogar 78\% der indischen Facebook Nutzer männlich. 
+\medskip
+Das rührt aber auch daher, dass auf Facebook keiner der Nutzer den Bildungsstand hinterfragt, den man angibt, was zu der Normalität führte, seinen Lebenslauf besser darzustellen, als er eigentlich ist. Es sei nicht selten, sich mehr als einen Account zu machen und auf den Zweitaccounts Profilbilder von Bollywood-Stars, oder anderen Berühmtheiten zu benutzen,
```

Comment:
Ich verstehe nicht ganz wie die erhöhte Anzahl männlicher Benutzer mit den Fakeaccounts zusammenhängen 
(`das rührt aber auch daher..`.)

Reply:
Irrelevant fürs Thema

Reply:
Naja in der Studie wird besagt, es sei gängig mehr als ein Account zu besitzen und dem Fake Account die Identität eines Celebrities zu geben. Und da es hauptsächlich männliche Inder machen entsthehen dementsprechend noch mehr männl. Accounts. 
Aber was ich mir aber auch vorstellen kann, ist, dass einige Inder sich auch weibliche Accounts erstellt haben or whatever, vlt nehme ich die Aussage einfach raus

Reply:
Wenn das die Aussage des Paragraphen sein soll, dann ist das beim Lesen nicht rübergekommen :)

Reply:
`dass auf Facebook keiner der Nutzer den Bildungsstand hinterfragt, den man angibt, was zu der Normalität führte, seinen Lebenslauf besser darzustellen, als er eigentlich ist. ` das ist aber irrelevant.

Wenn du sagen willst das die Menge der Accounts durch 2. Accounts hochgetrieben wird sag das auch